ng2-components/ng2-alfresco-documentlist/src/components/document-list.component.spec.ts

```diff
@@ -2,14 +2,14 @@
  * @license
  * Copyright 2016 Alfresco Software, Ltd.
  *
- * Licensed under the Apache License, Version 2.0 (the "License");
```

Comment:
No changed was added on the license header, i can change it, but checking on master branch other components have the double quote, is it ok?